modules/ww-shapemapper/README.md

```diff
@@ -0,0 +1,191 @@
+# ww-shapemapper Module
+
+[![Project Status: Prototype – Useable, some support, open to feedback, unstable API.](https://getwilds.org/badges/badges/prototype.svg)](https://getwilds.org/badges/#prototype)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+A WILDS WDL module for ShapeMapper 2, a tool for analyzing RNA structure probing data to determine nucleotide flexibility and secondary structure from SHAPE-MaP and related chemical probing experiments.
+
+## Overview
+
+ShapeMapper analyzes data from SHAPE (Selective 2'-Hydroxyl Acylation analyzed by Primer Extension) and related RNA structure probing methods. It processes paired-end sequencing data from chemically modified and untreated control samples to generate per-nucleotide reactivity profiles that reveal RNA secondary structure.
+
+This module provides a standardized WDL interface to ShapeMapper 2 for integration into larger bioinformatics workflows.
+
+## Module Structure
+
+This module is part of the [WILDS WDL Library](https://github.com/getwilds/wilds-wdl-library) and follows the standard WILDS module structure:
+
+- **Main WDL file**: `ww-shapemapper.wdl` - Contains task definitions for the module
+- **Test workflow**: `testrun.wdl` - Demonstration workflow for testing and examples
+- **Documentation**: This README with usage examples and parameter descriptions
+
+## Available Tasks
+
+### `run_shapemapper`
+
+Run ShapeMapper to analyze RNA structure probing data and generate reactivity profiles.
+
+**Inputs:**
+- `sample_name` (String): Name identifier for the sample
+- `target_fa` (File): FASTA file containing the target RNA sequence(s)
+- `modified_r1` (File): R1 FASTQ file from modified (chemically treated) sample
+- `modified_r2` (File): R2 FASTQ file from modified (chemically treated) sample
+- `untreated_r1` (File): R1 FASTQ file from untreated control sample
+- `untreated_r2` (File): R2 FASTQ file from untreated control sample
+- `primers_fa` (File, optional): FASTA file containing primer sequences for trimming
+- `min_depth` (Int, default=5000): Minimum read depth required for calculating reactivity
+- `is_amplicon` (Boolean, default=false): Set to true if data is from amplicon sequencing
+- `cpu_cores` (Int, default=2): Number of CPU cores allocated for the task
+- `memory_gb` (Int, default=8): Memory allocated for the task in GB
+
+**Outputs:**
+- `output_tar` (File): Compressed tarball of the ShapeMapper output directory
```

Comment:
```suggestion
- `output_tar` (File): Compressed tarball of a folder containing ShapeMapper outputs
```
I would clarify since ShapeMapper does not produce a directory itself